### ISSUE TITLE
llama loss function

### ISSUE THREAD

**Issue by eating-chen** [tloen/alpaca-lora#520]
When I see llama paper, I see they use perplexity as loss function
Is alpaca-lora loss perplexity or cross entropy when I choose llama model?
Thank you~

**Comment by ElleLeonne** [comment]
Perplexity is just exponential Cross Entropy. And it's usually used as an evaluation metric rather than a loss function.

Which is to say, I wouldn't worry too much about the distinction.
